quipucords/scanner/satellite/connect.py

```diff
@@ -75,27 +75,32 @@ def run(self, manager_interrupt):
                     % self.source.name
                 return error_message, ScanTask.FAILED
         except SatelliteAuthException as auth_error:
-            error_message = 'Satellite error encountered: %s. ' % auth_error
+            error_message = 'Satellite auth error encountered: %s. ' \
```

Comment:
Not a blocker but you could move these into a common messages file since they're used in multiple places.

Reply:
True